src/elements/about-organizer-block.js

```diff
@@ -46,15 +46,15 @@ class AboutOrganizerBlock extends ReduxMixin(PolymerElement) {
       }
     </style>
 
-    <div class="container" layout="" horizontal="">
-      <div layout="" horizontal="" center-center="" flex="" hidden\$="[[viewport.isPhone]]">
+    <div class="container" layout horizontal>
+      <div layout horizontal center-center flex hidden\$="[[viewport.isPhone]]">
```

Comment:
`hidden\$=""` is that a new way of handling properties or there should be `hidden$=`?

Reply:
No. `modulizer` escaped a bunch of `$`s and I haven't fixed them yet.